docs/application/dotnet/guides/app-management/app-controls.md

```diff
@@ -506,8 +499,11 @@ application requesting a launch of your application must either specify
 your application ID (for an explicit launch) or have the same operation
 value and applicable URI and MIME type information (for an implicit
 launch).
+You can define privileges to restrict launching your application. To launch
+your application, the caller application has to request the appropriate permission
+by adding privileges in the 'tizen-manifest.xml' file.
```

Comment:
Please rephrase to:

You can define privileges to restrict your application from launching. To launch your application, the caller application must request for launch permission by defining privileges in the 'tizen-manifest.xml' file.

Reply:
done